neo4j/test-integration/dbconn_test.go

```diff
@@ -22,6 +22,8 @@ package test_integration
 import (
 	"context"
 	"crypto/rand"
+	"fmt"
+	"github.com/neo4j/neo4j-go-driver/v5/neo4j/db"
```

Comment:
Why did the driver dependency move up here? :detective:

Reply:
something something leftover something

Reply:
I don't think this file changes very often, so maybe `go fmt` behavior changed a bit since last time?